modules/fscloud/variables.tf

```diff
@@ -3,50 +3,50 @@
 ##############################################################################
 
 variable "resource_group_id" {
-  description = "ID of resource group to use when creating the event stream instance"
+  description = "The resource group ID to use when creating the Event Streams instance."
```

Comment:
Shouldn't this be `Event Notifications instance` instead?

Reply:
correct, this is the Event Notifications module

Reply:
Made the update.